### PR TITLE
Add `remoteUserUIDIgnorePattern` property to ignore pattern (eg */nod…

### PR DESCRIPTION
**Pull request by edgar971**
Add `remoteUserUIDIgnorePattern` property to ignore pattern (eg */node_modules) in chown script (updateUID.Dockerfile)

| Property | Type | Description |
|----------|------|-------------|
| `remoteUserUIDIgnorePattern` | string | **Optional**  The regex being used inside the `updateUID.Dockerfile` script running the `chown` command used when `updateRemoteUserUID` is set to `true`.  Useful for skipping unnecessary UID/GID changes. |

Co-authored-by: erinwitmer <ecwitmer@gmail.com>
Co-authored-by: Payton J. Jones <paytonjjones@gmail.com>

### CODE REVIEW

File: scripts/updateUID.Dockerfile [hunk 1]
```diff
@@ -26,7 +27,7 @@ RUN eval $(sed -n "s/${REMOTE_USER}:[^:]*:\([^:]*\):\([^:]*\):[^:]*:\([^:]*\).*/
 		if [ "$OLD_GID" != "$NEW_GID" ]; then \
 			sed -i -e "s/\([^:]*:[^:]*:\)${OLD_GID}:/\1${NEW_GID}:/" /etc/group; \
 		fi; \
-		chown -R $NEW_UID:$NEW_GID $HOME_FOLDER; \
+	find $HOME_FOLDER -path IGNORE_PATTERN -prune -o -print0 | xargs -0 chown $NEW_UID:$NEW_GID; \
```
Comment (chrmarti) [inline scripts/updateUID.Dockerfile:30]: Missing `$` in front of `IGNORE_PATTERN`.

**Review by chrmarti — CHANGES_REQUESTED** [review]
Thanks for your PR! Left a comment. Please also check the above comment about the Contributor License Agreement. Thanks.

File: scripts/updateUID.Dockerfile (hunk 1 above)
Comment (edgar971) [reply scripts/updateUID.Dockerfile]: Added! Thanks!

**Comment by edgar971** [comment]
@microsoft-github-policy-service agree

**Pull request closed without merging**